spec/features/lessons/update_spec.rb

```diff
@@ -8,14 +8,17 @@
   # ACCEPTANCE CRITERIA
   # * As a leader of the course, I can update lessons
   # * So that learners can see changed details.
+  # * When lesson is updated, lesson no. stays unchanged.
   # * If I am a learner, I cannot see the update button.
 
   let!(:leader) { create :user }
   let!(:learner) { create :user }
   let!(:user) { create :user }
   let!(:course) { create :course_with_lessons }
-  let!(:leader_enrollment) { create :enrollment, role: 'leader', user: leader, course: course }
-  let!(:learner_enrollment) { create :enrollment, role: 'learner', user: learner, course: course }
+  let!(:leader_enrollment) { create :enrollment, role: 'leader',
+    user: leader, course: course }
```

Comment:
Align the elements of a hash literal if they span more than one line.<br>Expression at 19, 34 should be on its own line.